README.md

```diff
@@ -15,6 +15,11 @@ on how to do that, including how to develop and test locally and the versioning
 
 _Note: 1.28.0 and later require Gradle 7_
 
+### 1.35.0-SNAPSHOT
```

Comment:
Before merge, this should be changed to 1.35.0 and the release date updated to the date of the merge.